### ISSUE TITLE
FromScratch stays in background

### ISSUE THREAD

**Issue by andreapx** [Kilian/fromscratch#86]
FromScratch is a labour of love. Be nice and I'll listen to you! :) Will do!!! :)

Hi,
I've installed FromScratch on a new PC, here I can't use it.
When I open it I see the new icon in the task bar like it got open under the window I'm using, and in effect, if I click on it, the foreground window loose focus, but FromScratch doesn't appear.
How could I solve this?
Thanks!!!

**Comment by Kilian** [comment]
Which operating system are you using?

**Comment by andreapx** [comment]
Windows 10 Pro

**Comment by andreapx** [comment]
I've just discovered the problem: I've 2 display, and FromScratch was opening on the right one while the "main" one for Windows was the left one.
I've setup the display on the right to be the main one and I guess that from that moment I wasn't able to use it.
In the "windowstate" file I had x at 2000 pixels that would be ok if the pixel count started from the screen on the left (FromScratch would have been on the beginning of the right one), but when I changed the main screen to be the right one, the pixel count started from that one and the pixel 2000...well, it's outside the screen!
Now if I move FromScratch on the left screen, in the "windowstate" file i get a negative x.
Problem solved!
